tests/unit/backend/utils/attachment.spec.ts

```diff
@@ -0,0 +1,271 @@
+import { ObjectId } from 'bson'
+import { readFileSync } from 'fs'
+import { cloneDeep, merge } from 'lodash'
+
+import {
+  addAttachmentToResponses,
+  areAttachmentsMoreThan7MB,
+  attachmentsAreValid,
+  handleDuplicatesInAttachments,
+  mapAttachmentsFromParsedResponses,
+} from 'src/app/utils/attachment'
+import {
+  BasicField,
+  IAttachmentResponse,
+  ISingleAnswerResponse,
+} from 'src/types'
+
+const validSingleFile = {
+  filename: 'govtech.jpg',
+  content: readFileSync('./tests/unit/backend/resources/govtech.jpg'),
+  fieldId: String(new ObjectId()),
+}
+
+const invalidSingleFile = {
+  filename: 'invalid.py',
+  content: readFileSync('./tests/unit/backend/resources/invalid.py'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipWithValidAndInvalid = {
+  filename: 'invalidandvalid.zip',
+  content: readFileSync('./tests/unit/backend/resources/invalidandvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipNestedInvalid = {
+  filename: 'nested.zip',
+  content: readFileSync('./tests/unit/backend/resources/nestedInvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipNestedValid = {
+  filename: 'nestedValid.zip',
+  content: readFileSync('./tests/unit/backend/resources/nestedValid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipOnlyInvalid = {
+  filename: 'onlyinvalid.zip',
+  content: readFileSync('./tests/unit/backend/resources/onlyinvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const zipOnlyValid = {
+  filename: 'onlyvalid.zip',
+  content: readFileSync('./tests/unit/backend/resources/onlyvalid.zip'),
+  fieldId: String(new ObjectId()),
+}
+
+const MOCK_ANSWER = 'mockAnswer'
+
+describe('attachmentsAreValid', () => {
+  it('should return true when given a single valid file', () => {
+    return expect(attachmentsAreValid([validSingleFile])).resolves.toBe(true)
```

Comment:
oh, can be in regards to yuanruo's point
```suggestion
    await expect(attachmentsAreValid([validSingleFile])).resolves.toBe(true)
```

Reply:
I think his point was more to avoid using `.resolves` and `.rejects`, something like this:
```
const isValid = await attachmentsAreValid([validSingleFile])
return expect(isValid).toBe(true)
```

Reply:
oh ya! i guess this is indeed clearer